Wrapping/Generators/CastXML/CMakeLists.txt

```diff
@@ -290,7 +290,11 @@ macro(itk_wrap_module_castxml library_name)
   get_directory_property(include_dir_list INCLUDE_DIRECTORIES)
   list(REMOVE_DUPLICATES include_dir_list)
 
-  get_directory_property(compile_definition_list COMPILE_DEFINITIONS)
```

Comment:
Can we still keep this compile definition list around, then append the definitions for `${WRAPPER_LIBRARY_SOURCE_DIR}/..`? These compile definitions may be needed for another use case.

Reply:
sure!